cmake/leatherman.cmake

```diff
@@ -273,7 +277,11 @@ macro(gettext_compile dir inst)
                 DEPENDS ${fpath})
             add_custom_target(${lang}-${PROJECT_NAME} DEPENDS ${mo})
             add_dependencies(translations ${lang}-${PROJECT_NAME})
-            install(FILES ${mo} DESTINATION "${inst}/${lang}/LC_MESSAGES")
+            if(LEATHERMAN_LOCALE_VAR AND LEATHERMAN_LOCALE_INSTALL)
+                install(FILES ${mo} DESTINATION "${LEATHERMAN_LOCALE_VAR}/${LEATHERMAN_LOCALE_INSTALL}/${lang}/LC_MESSAGES")
```

Comment:
I think we need to take LEATHERMAN_LOCALE_INSTALL into account somehow, but either need to use CMAKE_INSTALL_PREFIX or a new variable to determine where to put the files for packaging.